### ISSUE TITLE
Breaking relations; mismapping streets

### ISSUE THREAD

**Issue by naoliv** [KaartGroup/Brazil#35]
I am really unsure how some streets are being mapped.
For example, here https://www.openstreetmap.org/changeset/66808797 there are no IBGE tiles, yet the source has `IBGE map of urban areas`

There are some major problems there: boundary relations were broken (which indicates that you are not validating the data before uploading); also, streets were traced where clearly there are no streets (which also indicates that you are not taking a look at the satellite imagery)

From where are you tracing the streets, please?
It feels that there is some kind of (wrong) automation in such editions and that you are also using data from somewhere else.

Likewise here https://www.openstreetmap.org/changeset/66836860

Again, there are no IBGE tiles there and you clearly mismapped the streets.

Could somebody explain the workflow that is being applied for such editions, please?

Is somebody verifying and validating them?

**Comment by garylancelot** [comment]
@naoliv this was also addressed with the team a few minutes ago this morning and they will be following up with you as soon as they come in today. Thank you again for your attention to these matters! Good map data is our first priority and we want to make sure we are catching any mistakes that are made.

**Comment by naoliv** [comment]
Hi, @garylancelot 

Something is still very wrong and strange with some of your team editions.
For example, here https://www.openstreetmap.org/changeset/67865528 we have a lot of street added at the midlle of the water!

Here https://www.openstreetmap.org/#map=16/-22.9472/-43.0709 at OSM we have right now this:

![](https://user-images.githubusercontent.com/982338/54501090-ed819c00-4901-11e9-98a3-d797134b78e9.png)

Taking a look with the IBGE Map of Urban Areas as the background we don't have anything there:

![](https://user-images.githubusercontent.com/982338/54501091-ee1a3280-4901-11e9-9f58-a31e4abf3fb8.png)


And with a sattelite layer:
![](https://user-images.githubusercontent.com/982338/54501092-ee1a3280-4901-11e9-8c0b-a3ce78d9b86f.png)

It seems that the sattelite imagery is not being used (I guess that it's very clear that who is editing is not even taking a look at it)

Edit: it seems that somehow the editor did a `Ctrl+C` and `Ctrl+V` and pasted/duplicated the streets at the wrong location. Perfectly understandable.

But two things still seem to be missing at the workflow: validate the data (JOSM would have catched the crossing streets) and checking the edition with a satellite imagery.

**Comment by kianaziola** [comment]
Thank you for bringing this to our attention. The problem has been resolved along with other errors in the same area. 
In reviewing the edits in question we discovered that the added roads were in fact duplicates of existing road ways in that same area. The editor that produced these errors has been spoken with and is now aware of the issues they caused. These edits likely resulted from a mistake in hotkey combinations, which we learned while trying to duplicate the issue. Thank you again and we look forward to working with you in the future.
